plugins/sfThemePlugin/config/sfThemePluginConfiguration.class.php

```diff
@@ -1,21 +0,0 @@
-<?php
```

Comment:
Please make sure to mention you removed this dead plugin in your commit description